app/components/elements/tables/row_component.rb

```diff
@@ -5,10 +5,11 @@ module Tables
     # Component for rendering a table row.
     class RowComponent < ApplicationComponent
       renders_many :cells
+      renders_many :items
 
       def initialize(label: nil, values: [], id: nil)
         @label = label
-        # Provide either values or cells.
+        # Provide either values, cells, or items.
```

Comment:
Can you expand this comment a bit to explain what each of them are?

Reply:
Added some more description.